CMakeLists.txt

```diff
@@ -519,11 +541,10 @@ function(add_picolibc directory variant target_triple flags qemu_params variant_
             picolibc_test(${test})
         endforeach()
         # Test disabled due to duplicated symbol name div in test file and in picolibc
-        # picolibc_test(rounding-mode ${picolibc_SOURCE_DIR}/test/rounding-mode-sub.c)
+        # picolibc_test(rounding-mode)
         # Test disabled due to fail on armv8.1m.main_hard_nofp_mve
-        # picolibc_test(try-ilp32 ${picolibc_SOURCE_DIR}/test/try-ilp32-sub.c)
+        # picolibc_test(try-ilp32)
         add_dependencies(check-picolibc check-picolibc_${variant})
-
```

Comment:
Avoid whitespace changes on lines that aren't connected to the main change you're making

Reply:
Done.